include/ppx/scene/scene_material.h

```diff
@@ -190,6 +199,24 @@ class ErrorMaterial
 
 // -------------------------------------------------------------------------------------------------
 
+// Debug Material
+//
+// Implementations should render position, tex coord, normal, and tangent as color output.
+//
+class DebugMaterial
+    : public scene::Material
+{
+public:
+    DebugMaterial() {}
+    virtual ~DebugMaterial() {}
+
+    virtual std::string GetIdentString() const override { return PPX_MATERIAL_IDENT_DEBUG; }
```

Comment:
`override` is not strictly needed here, right? Given that you're actually overriding the function.  But it doesn't hurt, I guess.

Reply:
Missing override will cause build failure on Android.